agent/config/parse_linux.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 func parseGMSACapability() BooleanDefaultFalse {
-	envStatus := utils.ParseBool(os.Getenv(envGmsaEcsSupport), true)
+	envStatus := utils.ParseBool(os.Getenv(envGmsaEcsSupport), false)
```

Comment:
Why this change?

Reply:
by default it should be false, since customer need to use ECS_GMSA_SUPPORT env, variable, last time we missed it